src/components/pacbio/PacbioPoolAliquotEdit.vue

```diff
@@ -217,4 +228,10 @@ const isValidationExists = (attribute) => {
     return !fieldsThatRequireValidation.value[attribute]
   }
 }
+/*
+  This is to display any volume error for a library request on mounting component in case 
+  the auto-filled volume (original volume of library) is greater than the available volume
+ */
+aliquot.value.source_type === 'Pacbio::Library' &&
```

Comment:
Could this be done as part of selectRequest in the store instead? i.e as you select a request we validate it to check the volume and add the error.